packages/react-composites/tests/browser/chat/fixture.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PARTICIPANT_NAMES } from '../config';
```

Comment:
`PARTICIPANT_NAMES` have been removed from config. How is this working?

Reply:
Oops. Don't now how this leaked in. That variable is actually deleted from `../config`